flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -446,6 +446,31 @@ class Table(
   }
 
   /**
+    * Intersect two [[Table]]s with duplicate records removed. Intersect returns rows only
+    * from the left table that are identical to a row in the right table.
+    * Similar to an SQL INTERSECT. The fields of the two union operations must fully overlap.
```

Comment:
"The fields of the two union operations must fully overlap." -> "The fields of the two intersected tables must fully overlap."